tests/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 ### Setup ###
-# Don't run tests when we're building specific bindings...
-if(OSSIA_PD OR OSSIA_MAX OR OSSIA_PYTHON OR OSSIA_NODEJS OR OSSIA_JAVA OR OSSIA_UNITY3D)
+# Don't run tests on CI when we're building specific bindings...
+if(OSSIA_CI)
```

Comment:
hm I remember disabling that at some point but can't remember why

Reply:
disabling what ? the `if(OSSIA_CI)` or the tests themselves ?
I'm sure it's a good idea to disable test on CI for those configurations
but then for local build we should be able to enable test